p5/core/api.py

```diff
@@ -14,8 +14,16 @@
 from .material import normal_material, basic_material, blinn_phong_material
 from .light import ambient_light, directional_light, point_light, light_falloff, light_specular
 
-
 def push():
+    """	The push() function saves the current drawing style settings and transformations"""
```

Comment:
wasnt this there before?

Reply:
I wrongly named `pushStyle` as `push` back then when I added the support for processing APIs.

Here is the [link](https://processing.org/reference/push_.html) for the `push()` API that was missing in p5py.
There is `push_matrix` and `push_style` present in p5py but no `push()` that stores both **transformation** and **style** changes.

`push` in p5.js [link](https://p5js.org/reference/#/p5/push)

Found this error after trying to implement this example.

```
fill(255);
rect(0, 0, 50, 50);  // White rectangle

push();
translate(30, 20);
fill(0);  
rect(0, 0, 50, 50);  // Black rectangle
pop();  // Restore original settings

fill(100);  
rect(15, 10, 50, 50);  // Gray rectangle
example pic
ellipse(0, 50, 33, 33);  // Left circle

push(); 
strokeWeight(10);
fill(204, 153, 0);
ellipse(50, 50, 33, 33);  // Middle circle
pop();  // Restore original settings

ellipse(100, 50, 33, 33);  // Right circle
```
The results were different in p5py and p5.js.